### ISSUE TITLE
High-speed speedup tubes connect sideways with their "slow" ends

### ISSUE THREAD

**Issue by v1993** [RS485/LogisticsPipes#1644]
Probably a single screenshot will describe the issue better than any amount of words:

![2021-08-16_17 57 08](https://user-images.githubusercontent.com/16255906/129584417-06cc7da2-6b26-4f15-bc47-316f2a433c64.png)

Items don't actually travel sideways there, so red "error" indication is wrong - there is no issue here.

**Comment by v1993** [comment]
Actually - never mind, there is another problem here too. While items never travel sideways, system treats that connection as existing and tries to send items belonging to the right pipe into the left one.

**Comment by v1993** [comment]
Actually - it's worse than that. System treats two nearby high-speed tubes as connected (even if they are not) in general, causing hard to debug issues if two of them are placed near each other.

**Comment by v1993** [comment]
There are two separate issues by this point, apparently:

1. One seen in the screenshot with high-speed tubes wrongly connecting their ends
2. System not checking that high-speed tubes are connecting when calculating reachability

Should I open a separate issue to track the second problem?

**Comment by KorewaLidesu** [comment]
Use Disconnect upgrade to separate end of high-speed tube
The end of high-speed tube is unrouted pipe, which cause inflict issue

**Comment by v1993** [comment]
I had no idea disconnection upgrade can be used there! Still, IMO this does stay as a valid issue.

**Comment by KorewaLidesu** [comment]
>Still, IMO this does stay as a valid issue.

`The end of high-speed tube is unrouted pipe, which cause inflict issue`

**Comment by v1993** [comment]
Yes, I understood what you said. Yes, this still seems like a valid bug to me, because items *don't actually travel sideways there, ever*. This is *not* how usual unrouted pipes work, they just choose destination randomly. Your explanation shows *why this bug happens*, not *why this is not a bug*.

Second issue of system treating nearby pipes as connected extends to all high-speed tubes, not just their slow-end sections.
